### PR TITLE
Remove commons-lang dependency

### PR DESCRIPTION
**Pull request by jburel**
Remove usage of commons-lang: insight only
This was generating an error when using the omero-ij plugin in Fiji
see https://trac.openmicroscopy.org.uk/ome/ticket/12713

To test this PR:
 * Make sure the server starts

Fiji Test
 * Check that there is a jar under the jars folder commons-lang3-3.x
 * Add the omero-ij plugin to Fiji plugins folder. 
 * Check that you can connect to OMERO using the plugin

This PR replaces https://github.com/openmicroscopy/openmicroscopy/pull/3474
--no-rebase

### CODE REVIEW

**Comment by jburel** [comment]
One day under breaking but this time everything should work.

**Comment by sbesson** [comment]
https://ci.openmicroscopy.org/job/OMERO-5.1-breaking-trigger/286/ and downstream jobs coped well this this change.. Removing `breaking` label

**Comment by dominikl** [comment]
It works on windows, but on OSX I still get the `java.lang.NoSuchFieldError: IS_OS_WINDOWS_7` error.
I used latest FIJI from the FIJI downloads site and today's breaking build [OMERO.insight-ij-5.1.0-m3-1077-add15e3-ice35-b287.zip](https://ci.openmicroscopy.org/view/Breaking/job/OMERO-5.1-breaking-build/ICE=3.5,jdk=8_LATEST,label=trout/lastSuccessfulBuild/artifact/src/target/OMERO.insight-ij-5.1.0-m3-1077-add15e3-ice35-b287.zip) for testing. I also noticed that in omero-ij plugin's lib folder still is commons-lang.jar, that should have been removed, right?

**Comment by jburel** [comment]
@dominikl: are you sure you are using the latest code, since we no longer use the static field IS_OS_WINDOWS_7

**Comment by dominikl** [comment]
Oh, it looks like the PR was excluded from the breaking build: [GIT_INFO](https://ci.openmicroscopy.org/view/Breaking/job/OMERO-5.1-breaking-build/ICE=3.5,jdk=8_LATEST,label=trout/lastSuccessfulBuild/artifact/src/target/GIT_INFO)

**Comment by jburel** [comment]
@dominikl: thanks for the initial review. I have removed the jar.

**Comment by sbesson** [comment]
@dominikl: thanks. I just realized this PR had been marked as `breaking` after yesterday's suite was started. Will mark as `breaking` for this afternoon. We can re-include for tomorrow's merge build if everything works as expected.

**Comment by dominikl** [comment]
Alright, then I'll review the PR again tomorrow.

**Comment by dominikl** [comment]
Server and Insight working fine; omero-ij plugin works, too. There are no more references to `org.apache.commons.lang.*` on the client side (there are on the server side, but that's expected, isn't it?). Just the commons-lang.jar is still included in the omero-ij plugin (libs directory), but after a manual deletion, the plugin still works.

File: components/insight/SRC/org/openmicroscopy/shoola/util/CommonsLangUtils.java [hunk 1]
```diff
@@ -0,0 +1,368 @@
+/*
+ *------------------------------------------------------------------------------
+ *  Copyright (C) 2015 University of Dundee. All rights reserved.
```
Comment (joshmoore) [inline components/insight/SRC/org/openmicroscopy/shoola/util/CommonsLangUtils.java]: Are the methods copied from commons-lang? If so, a different copyright header might be appropriate.

Comment (jburel) [inline components/insight/SRC/org/openmicroscopy/shoola/util/CommonsLangUtils.java]: Was planning to do that.

**Comment by jburel** [comment]
@dominikl: server-side refs are expected.

**Comment by joshmoore** [comment]
The new header works for me. @jburel : you're looking into omero-ij's libs directory?

**Comment by jburel** [comment]
The lib directory is a general problem of the build (not specific to omero-ij) gets included.
I will investigate that issue and clean up the ivy file.
I can do that in this PR or another one (my preference).

**Pull request merged**